Makefile

```diff
@@ -11,9 +11,23 @@ export BUILD_INFO := $(shell ./build-tools/version-tool build-info)
 
 GO_BUILD_FLAGS=-v -ldflags "-extldflags \"-static\" -X main.version=$(BUILD_VERSION) -X main.buildInfo=$(BUILD_INFO)"
 
-# Allow users to pass in BASE_OS build options (alpine or rhel7)
+# Allow users to pass in BASE_OS build options (alpine or rhel7) 
 BASE_OS ?= alpine
 
+# This is for builds not triggered through Travis CI 
+ifndef LICENSE_STRICT
+	LICENSE_STRICT=false
+endif
+
+# Flag for level of license approval check 
+# (default if no flag is passed is devel)
+LIC_FLAG=
```

Comment:
Is this actually doing anything?

Reply:
I am going to try  with this line removed and see if things work as expected.